_data/apis.yml

```diff
@@ -195,6 +199,7 @@
 
 - name: Language Weaver
   id: language-weaver
+  other_names: [ SDL Language Weaver, RWS Language Weaver ]
```

Comment:
Maybe this is not necessary?

More like:

```
company: rws
```

And then in the RWS company data, we can include SDL.

But not a blocker, can be done later.